src/index.js

```diff
@@ -230,6 +230,23 @@ class App extends React.Component {
             const y = e.clientY - e.target.offsetTop;
             const element = newElement(this.state.elementType, x, y);
 
+            let isDraggingElements = false;
+            const cursorStyle = document.documentElement.style.cursor;
+            if (this.state.elementType === "selection") {
+              isDraggingElements = elements.some(el => {
+                if (el.isSelected) {
+                  const minX = Math.min(el.x, el.x + el.width);
```

Comment:
I have a helper getElementAbsoluteX1 that does that. Probably want to share the logic here and the bound check test